python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/management-security.j2

```diff
@@ -34,15 +34,15 @@
 
 ### Management Security SSL Profiles
 
-| SSL Profile Name | TLS protocol accepted | Certificate filename | Key filename | Cipher List | CRLs |
-| ---------------- | --------------------- | -------------------- | ------------ | ----------- | ---- |
+| SSL Profile Name | TLS protocol accepted | Certificate filename | Key filename | Cipher List | Cipher v1.0 | Cipher v1.3 | CRLs |
```

Comment:
I would rather we avoid have extra columns because both cannot be supported at the same time.

Maybe we can rename `Cipher List` to `Ciphers` and for post 4.32.0F have the cell looks like

```
v1.0 to v1.2: {{ ssl_profile.ciphers.v1_0 | arista.avd.default('-') }}
v1.3: {{ ssl_profile.ciphers.v1_3 | arista.avd.default('-') }}
```

Reply:
Fixed